src/routes/write-exercise/components/__tests__/MissingArticlePopover.spec.tsx

```diff
@@ -5,21 +5,24 @@ import labels from '../../../../constants/labels.json'
 import wrapWithTheme from '../../../../testing/wrapWithTheme'
 import MissingArticlePopover from '../MissingArticlePopover'
 
-// will be fixed in LUN-230
-// eslint-disable-next-line jest/no-disabled-tests
-describe.skip('MissingArticlePopover', () => {
+jest.mock('react-native-popover-view', () => ({
+  __esModule: true,
+  default: 'Popover',
+  PopoverPlacement: 'top'
+}))
+
+describe('MissingArticlePopover', () => {
   it('should show Popover when visible', () => {
     const { getByText } = render(<MissingArticlePopover setIsPopoverVisible={jest.fn()} isVisible={true} />, {
       wrapper: wrapWithTheme
     })
     expect(getByText(labels.exercises.write.feedback.articleMissing)).toBeDefined()
```

Comment:
but wouldn't his be always be defined, also if popover is invisible, as you wrote on the test below. So maybe we should check here also for the visible prop?

Reply:
you're right. @Taggotty and me tried some stuff but didn't find a better solution so far